WordPress/gutenberg
# Adds Aspect ratio control on Image blocks in Grids

## What?
Fixes: #62889 

## Why?
Currently, when adding an Image block within a Grid layout, the Aspect Ratio control is disabled. This can limit user's ability to set consistent visual proportions across images within the grid. Enabling the Aspect Ratio control allows users to maintain uniformity in image display.

## How?
Conditionally shows Aspect ratio control on grid layouts.

## Testing Instructions
1. Add a Grid block
2. Add few images with different aspect ratios to the Grid
3. Notice that the Aspect Ratio control is **visible** for the images

## Screenshots or screencast 
https://github.com/WordPress/gutenberg/assets/77401999/e6d7c76e-ca18-4744-aba1-44f4b1b91a52
